executor/partition_table_test.go

```diff
@@ -673,3 +673,97 @@ func (s *globalIndexSuite) TestIssue21731(c *C) {
 	tk.MustExec("drop table if exists p, t")
 	tk.MustExec("create table t (a int, b int, unique index idx(a)) partition by list columns(b) (partition p0 values in (1), partition p1 values in (2));")
 }
+
+func (s *testSuiteWithData) TestRangePartitionBoundariesBetweenM(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("create database TestRangePartitionBoundariesBetweenM")
+	tk.MustExec("use TestRangePartitionBoundariesBetweenM")
+	tk.MustExec(`CREATE TABLE t
```

Comment:
Please drop it if exists

Reply:
Updated the PR with `DROP TABLE IF EXISTS` and `DROP DATABASE IF EXISTS` too.